q2cli/tools.py

```diff
@@ -40,7 +40,12 @@ def export_data(input_path, output_path, output_format):
     import distutils
     result = qiime2.sdk.Result.load(input_path)
     if output_format is None:
-        result.export_data(output_path)
+        if isinstance(result, qiime2.sdk.Artifact):
+            output_format = result.format.__name__
+            result.export_data(output_path)
```

Comment:
Why is this line duplicated here and on L48? The result export doesn't appear to have anything to do with the `if/else` statement on L43-L48 --- can't it be moved out of the branching block?